src/main/org/codehaus/groovy/transform/trait/Traits.java

```diff
@@ -355,19 +355,19 @@ static String getSuperTraitMethodName(ClassNode trait, String method) {
      */
     @Retention(RetentionPolicy.RUNTIME)
     @Target(ElementType.METHOD)
-    public static @interface Implemented {}
+    public @interface Implemented {}
 
     /**
      * Internal annotation used to indicate that a method is a bridge method to a trait
      * default implementation.
      */
     @Retention(RetentionPolicy.RUNTIME)
     @Target(ElementType.METHOD)
-     public static @interface TraitBridge {
```

Comment:
The `static` modifier is redundant on inner interfaces. The bytecode will remain unchanged with or without the modifier so the code referenced should continue to work.